src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        ListOfColors[] colors = ListOfColors.values();
+        int indexOfColor = new Random().nextInt(colors.length);
+        return colors[indexOfColor].toString();
+    }
+
+    public int getRandomINum() {
+        return new Random().nextInt(101);
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes

think about making a class-level field:
```
private Random random;
```